compendium/ConfigurationAdmin/src/ConfigurationAdminImpl.cpp

```diff
@@ -199,6 +221,9 @@ namespace cppmicroservices {
 
       managedServiceFactoryTracker.Close();
       managedServiceTracker.Close();
+      if (configListenerTracker) {
```

Comment:
trackers can throw an exception on Close. Surround these 3 Close statements with try/catch so the destructor does not throw an exception and log the exception using the logger